CHANGELOG.next.asciidoc

```diff
@@ -74,6 +74,7 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...main[Check the HEAD dif
 - Fixed missing ZooKeeper metrics due compatibility issues with versions >= 3.6.0 {pull}30068[30068]
 - Fix Docker module: rename fields on dashboards. {pull}30500[30500]
 - Add back missing metrics to system/linux. {pull}30774[30774]
+- Allow named instances in mssql module. {issue}24076[24076] {pull}30859[30859]
```

Comment:
This line must not be here.

Reply:
Could be an important information for mssql users.